client/src/assets/bottomNavCheckHookIcon.svg

```diff
@@ -0,0 +1,4 @@
+<svg id="icon_action_check_circle_24px" data-name="icon/action/check_circle_24px" xmlns="http://www.w3.org/2000/svg" width="36" height="36" viewBox="0 0 36 36">
+  <rect id="Boundary" width="36" height="36" fill="none"/>
+  <path id="_Color" data-name=" ↳Color" d="M16,32A16,16,0,1,1,32,16,16.018,16.018,0,0,1,16,32ZM7.056,13.743h0L4.8,16l8,8L27.2,9.6,24.944,7.328,12.8,19.472,7.056,13.743Z" transform="translate(2 2)" fill="#f0eaea"/>
+</svg>
```

Comment:
You can minify svg files with the help of sites like: https://www.svgminify.com/

Reply:
Thanks for the link